### DISCUSSION TITLE
Calendar -> fullCellRender props -> Outdated Docs

### DISCUSSION THREAD

**Discussion by devcaco** [ant-design/ant-design#51535] [Q&A]
Hello. According to the current documentation for the calendar component, the fullCellRender and cellRender props should get parameters:

current: dayJS
today: dayJS
info: object

However this is not true. With the latest version, it only accepts date: dayJS and info: object.

Is this correct or am I missing something?

If so the documentations should be updated. Thanks

**Comment by petercat-assistant** [comment]
I found a related pull request titled "site(Calendar): add lunar calendar demo" in the Ant Design repository. However, it doesn't seem directly related to the issue you're experiencing with the `fullCellRender` and `cellRender` props.

It appears that this might be a new issue or a documentation oversight. I recommend reporting this discrepancy to the Ant Design team so they can update the documentation accordingly.

For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.

**Reply by devcaco** [reply]
Hi @petercat-assistant thanks for the quick reply.

How can I report it to the antd team? Should I create a new issue?

**Comment by petercat-assistant** [comment]
It seems there might be a misunderstanding. Let me clarify: the issue you're experiencing with the `fullCellRender` and `cellRender` props not matching the documentation could indeed be a documentation oversight or a change in the API that hasn't been reflected in the documentation yet.

If you believe this is a bug or an oversight, you might want to report it to the Ant Design team so they can investigate and update the documentation if necessary.

For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.

**Comment by RSS1102** [comment]
follow up #52137
